assets/css/default-calendar-common.css

```
@@ -0,0 +1,459 @@
+@font-face {
+	font-family: 'simple-calendar';
```

Stylelint (17.14.0)

[error] Expected no quotes around "simple-calendar" (font-family-name-quotes)

(font-family-name-quotes)